types/gbk.js/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for gbk.js 0.3
+// Project: https://github.com/cnwhy/GBK.js
+// Definitions by: mkchung <https://github.com/mkchung>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace gbkjs {
+    function encode(str: string): number[];
+    function decode(bytes: number[]): string;
+    namespace URI {
+      function encodeURI(str: string): string;
+      function decodeURI(str: string): string;
+      function encodeURIComponent(str: string): string;
+      function decodeURIComponent(str: string): string;
+    }
+}
+
+export default gbkjs;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default